src/common/src/catalog/mod.rs

```diff
@@ -94,21 +96,17 @@ pub fn default_key_column_name_version_mapping(version: &ColumnDescVersion) -> &
 
 /// For kafka source, we attach a hidden column [`KAFKA_TIMESTAMP_COLUMN_NAME`] to it, so that we
 /// can limit the timestamp range when querying it directly with batch query. The column type is
-/// [`DataType::Timestamptz`]. For more details, please refer to
+/// [`crate::types::DataType::Timestamptz`]. For more details, please refer to
 /// [this rfc](https://github.com/risingwavelabs/rfcs/pull/20).
 pub const KAFKA_TIMESTAMP_COLUMN_NAME: &str = "_rw_kafka_timestamp";
 
+/// RisingWave iceberg table engine will create the column `_risingwave_iceberg_row_id` in the iceberg table.
+///
+/// Iceberg V3 spec use `_row_id` as a reserved column name for row lineage, so if the table without primary key,
+/// we can't use `_row_id` directly for iceberg, so use `_risingwave_iceberg_row_id` instead.
 pub const RISINGWAVE_ICEBERG_ROW_ID: &str = "_risingwave_iceberg_row_id";
-pub fn is_system_schema(schema_name: &str) -> bool {
-    SYSTEM_SCHEMAS.iter().any(|s| *s == schema_name)
-}
-
-pub const ROWID_PREFIX: &str = "_row_id";
```

Comment:
Any idea why it was previously called "prefix"? Looks like there's no usage for appending something to it. cc @st1page

Reply:
https://github.com/risingwavelabs/risingwave/blame/87b8ffc265e4c5c3babdb79db1ef4ec8d1864dbc/src/frontend/src/catalog/mod.rs#L34-L43